WebDriverAgentLib/Categories/XCElementSnapshot+FBHelpers.m

```diff
@@ -122,6 +126,20 @@ - (BOOL)fb_framelessFuzzyMatchesElement:(XCElementSnapshot *)snapshot
     valuesAreEqual(self.placeholderValue, snapshot.placeholderValue);
 }
 
+- (BOOL)fb_hasMoreThanOneVisibleChildSnapshot
```

Comment:
You could make it `fb_hasMoreThanOneVisibleChildSnapshotOfType:`. It would really help to refactor scrolling as it will kill a lot of repetition